src/zoid/card-fields/component.jsx

```diff
@@ -289,6 +289,10 @@ export const getCardFieldsComponent : () => CardFieldsComponent = memoize(() : C
         },
 
         props: {
+            action: {
+                type: 'object',
+                required: true
```

Comment:
I was referencing this PR for config stuff and noted that this shouldn't be required, right? It's a bit of a confusing scenario because if no `createOrder`, etc. callbacks are provided, then it _is_ required. 🤔 hmm that's tricky.

Reply:
you're right, we'll have to use Zoid's `validate` prop to check validity for the action prop: https://github.com/krakenjs/zoid/blob/main/docs/api/prop-definitions.md#validate